Theme1/src/firsttheme/PalindromeCheck.java

```diff
@@ -0,0 +1,38 @@
+package firsttheme;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PalindromeCheck {
+    private static final Pattern pattern = Pattern.compile("\\W");
+    private static Matcher matcher;
+    public static int countOfPalindromes(String strings) {
+        int retval = 0;
+        if (strings.equals("")) return retval;
+        for(String string : strings.split(" ")) {
+            if(isPalindrome(string)) retval++;
+        }
+        return retval;
+    }
```

Comment:
Please add new line to separate method and properties declaration. By default use such approach:

public class PalindromeCheck {

    private static final Pattern pattern = Pattern.compile("\\W");
    private static Matcher matcher;

    public static int countOfPalindromes(String strings) {
      ....
    }

    public static int countOfPalindromes(String[] strings) {
       ...
    }